tools/installPluginVersion.php

```diff
@@ -55,6 +55,38 @@ function execute() {
 		$pluginVersion = $versionInfo['version'];
 		$versionDao = DAORegistry::getDAO('VersionDAO');
 		$versionDao->insertVersion($pluginVersion, true);
+
+		$pluginPath = str_replace('/version.xml', '', $this->_descriptor);
```

Comment:
Use `dirname($this->_descriptor);` -- or alternately maybe it would be better to provide the plugin path to the command line for this tool, rather than the `version.xml`.